config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-presubmits.yaml

```diff
@@ -48,6 +48,6 @@ presubmits:
         - runner.sh
         - /workspace/scenarios/kubernetes_bazel.py
         args:
-        - "--install=gubernator/test_requirements.txt"
+        - "--install=hack/test_requirements.txt"
```

Comment:
I think the more appropriate fix would be to remove this line, unless you have python you need to test in this repo.

Reply:
(even then, this was a hacky fix because gubernator isn't fully bazelified)